config/maps/testMap.txt

```diff
@@ -0,0 +1,7 @@
+N N N N N N N P N N N N N N N
```

Comment:
make sure to leave a documentation somewhere for the representations of these characters